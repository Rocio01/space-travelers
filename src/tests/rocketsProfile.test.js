import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import RocketsProfile from '../components/rocketsProfile';
import store from '../redux/configureStore';

test('matches RocketsProfile component snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <RocketsProfile />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
