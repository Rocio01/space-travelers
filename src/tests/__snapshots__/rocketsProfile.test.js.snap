// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches RocketsProfile component snapshot 1`] = `
<ul
  className="list-group"
/>
`;
